src/theory/sets/theory_sets_type_rules.cpp

```diff
@@ -41,25 +102,37 @@ TypeNode SetsBinaryOperatorTypeRule::computeType(NodeManager* nodeManager,
 {
   Assert(n.getKind() == Kind::SET_UNION || n.getKind() == Kind::SET_INTER
          || n.getKind() == Kind::SET_MINUS);
-  TypeNode setType = n[0].getType(check);
+  TypeNode setType = n[0].getTypeOrNull();
```

Comment:
Looked at the documentation for getTypeOrNull. It contains a typo in word `execption`. It would be nice if it is included in this PR.

Reply:
Done